rubocop.gemspec

```diff
@@ -42,6 +42,6 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('unicode-display_width', '~> 1.0', '>= 1.0.1')
 
   s.add_development_dependency('bundler', '~> 1.3')
-  s.add_development_dependency('rack')
+  s.add_development_dependency('rack', '>= 1.6.9', '< 2.0')
```

Comment:
Can you also add a todo here we need to update this once we drop support for Ruby 2.1?

Reply:
Sure thing, done.